src/api/app/components/action_seen_by_user_component.html.haml

```diff
@@ -2,12 +2,14 @@
   = render_icon_status
 
 - else
-  - button_text = 'Action seen'
+  - button_text = 'Seen'
 
   = link_to(toggle_request_path(action_id: @action.id),
-      class: 'btn btn-sm btn-outline-info',
+      class: 'btn btn-sm text-info',
       title: 'Whether the action has been marked "as seen" or not yet',
       method: :put, remote: true,
       'data-disable-with': tag.i(class: 'fa-solid fa-spinner fa-spin') + button_text) do
+    (
```

Comment:
@ncounter to me the brackets around the seen check mark make it look like a 'side' info. but it is an action. I would remove them.

Reply:
maybe reintroducing the outline, but very light? Since the `Action details` header is very strong the `Seen` checkmark is to much in the background. If I don't know its there I have to look twice in order to see it

Reply:
I see, thanks for the feedback. I think I agree with you it still misses some highlight. I'll bring back the outline (as it is a button in the end). :+1:

Reply:
I addressed it and updated the screenshot in the PR description